setup.py

```diff
@@ -19,6 +19,7 @@
         'console_scripts': [
             'filetracker = filetracker.shell:main',
             'filetracker-server = filetracker.servers.run:main',
+            'ftcachecleanerd = filetracker.cachecleaner:main',
```

Comment:
Po chwili namysłu i krótkiej wymianie zdań z innymi to całkiem niezły pomysł. Poprawię.